pystarport/buf.gen.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 managed:
   enabled: true
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/python:v3.19.1-1
+  - plugin: buf.build/protocolbuffers/python:v22.3
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.